aea/configurations/constants.py

```diff
@@ -23,6 +23,8 @@
 
 
 FETCHAI = "fetchai"
+ETHEREUM = "ethereum"
+COSMOS = "cosmos"
```

Comment:
Do we still need them?

Reply:
yes, as some parts of the framework use those identifiers. Without them, we'd have to import `<ledger-id>_crypto`.

Reply:
which parts?